bool left
bool front_left
bool front_right
bool right

int16 left_signal
int16 front_left_signal
int16 front_right_signal
int16 right_signal
